### PR TITLE
Update set-dhcpserverauditlog.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
Issue #531 ,
the parameter specifies limit for all the files, not for the single file, old description could be misleading

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Minor copy editing is needed for this PR. Thanks.

File: docset/windows/dhcpserver/set-dhcpserverauditlog.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Accept wildcard characters: False
 ```
 
 ### -MaxMBFileSize
-Specifies the maximum size of the audit log, in megabytes (MB).
+Specifies the maximum size on disk available for all DHCP service audit log files, in megabytes (MB).
```
Comment (JohanFreelancer9) [inline docset/windows/dhcpserver/set-dhcpserverauditlog.md]:
```suggestion
Specifies the maximum disk space available for all DHCP service audit log files, in megabytes (MB).
```

**Comment by e0i** [comment]
@MaratMussabekov Please incorporate suggested changes into your PR. Thanks.

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 ,
you recommendations were applied, thank you!

**Comment by MaratMussabekov** [comment]
Dear @e0i ,
the parameter "MaxMBFileSize" changes the value of registry keys: HKLM\SYSTEM\CurrentControlSet\Services\DhcpServer\Parameters\DhcpLogFilesMaxSize , which accroding to [this source](https://blogs.msdn.microsoft.com/anto_rocks/2006/11/02/all-about-dhcp-auditing/):

> Specifies the maximum **combined** size of one week's worth of Dynamic Host Configuration Protocol (DHCP) audit logs. If the audit logs exceed the size specified by this value, DHCP stops writing to the audit log until sufficient space becomes available.

Thank you

**Pull request merged**